Python/bytecodes.c

```diff
@@ -180,19 +180,19 @@ dummy_func(
             LOAD_FAST,
         };
 
-        inst(LOAD_FAST_CHECK, (-- value)) {
+        pure tier2manual inst(LOAD_FAST_CHECK, (-- value)) {
```

Comment:
I don't think we should introduce `tier2manual`.

TLDR, the sense I'm getting from upstream is that the optimizer is just a plug-and-play thing, where you can switch out optimizers. We should not introduce a keyword that ties the DSL to our optimizer implementation.

`pure` is a bit more general, where it can work for multiple things.

Reply:
Yea ok sure